package.json

```diff
@@ -145,7 +150,7 @@
     "coveralls": "^3.0.2",
     "cross-env": "^5.2.0",
     "dts-bundle-generator": "^1.6.1",
```

Comment:
seems like most criticals are related to `conventional-changelog-cli` and it's dependencies. I dunno whether updating this has an effect on the dts-bundle-generator but if not this might be a good time to update those as well and get rid of the npm audit proposed issues

Reply:
Other than the jest-related dependencies every other dependencies are updated.